include/tvm/target/target.h

```diff
@@ -54,7 +54,7 @@ class TargetNode : public Object {
   /*! \brief Keys for this target */
   Array<String> keys;
   /*! \brief Collection of attributes */
-  Map<String, ObjectRef> attrs;
+  Map<String, ObjectRef> attrs;  // TODO(@electriclilies): Unify with DictAttrs on IRModule
```

Comment:
Target has their own attributions, for example, compute version for cuda devices. Would you like to elaborate the reason why we need to unify we need to unify this attrs into IRModule?

Reply:
Target attrs is different from the IRModule attrs. They contain attributes about a target(cuda) that can be shared across IRModule or functions.

Please double check if we really want to remove the target.attrs(my guess is that it is unlikely) would be great to confirm with the original authors of the target (e.g. @zxybazh )

Reply:
I'm not sure if this is needed, I added it as part of my initial read through of this section of the codebase. I'll remove it and if it's needed we can discuss in a later PR